compiler/luci/pass/src/FuseGeluPass.cpp

```diff
@@ -179,13 +273,13 @@ luci::CircleGelu *FuseGelu::create_gelu(loco::Graph *graph)
   gelu->features(_p->_ifm);
   // TODO Support approximate = True pattern
   gelu->approximate(false);
-  gelu->name(_p->_mul_half->name() + "_gelu");
+  gelu->name(_p->_pattern_last_node->name() + "_gelu");
```

Comment:
This change doesn't seem to be related with `GeluPattern2`, can you describe how it is?

Reply:
Removed it